sklearn/tests/test_cross_validation.py

```diff
@@ -1005,26 +1014,6 @@ def test_shufflesplit_reproducible():
     assert_array_equal(list(a for a, b in ss), list(a for a, b in ss))
 
 
-def test_safe_split_with_precomputed_kernel():
```

Comment:
Ah I guess they are testing methods/functions starting with underscore which you didn't feel necessary to keep in sklearn.cross_validation. Fair enough.